api/policies/v1/policygroup_validation.go

```diff
@@ -138,3 +140,40 @@ func validatePolicyGroupExpressionField(policyGroup PolicyGroup) *field.Error {
 
 	return nil
 }
+
+// validatePolicyGroupMembersTimeouts checks the timeouts so that:
+//   - the group's timeoutSeconds is not greater than the Kubernetes webhook max timeout (30s).
+//   - each member's timeoutEvalSeconds is not greater than the group's
+//     timeoutSeconds nor the Kubernetes webhook max timeout (30s).
+func validatePolicyGroupMembersTimeouts(policyGroup PolicyGroup) field.ErrorList {
```

Comment:
I think this is good enough, but if we wanted to be fancy we could also ensure the sum of all the member's timeoutEvalSeconds is not going to be greater than the group's timeoutSeconds nor the k8's max timeout value

Reply:
Good point. Rebased and added that logic, plus the unit tests that check it.

Reply:
thanks!